elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,449 @@
+  --_secondary-background-color: transparent;
+  --_secondary-border-color: #2578bb; /* WARNING: not a token value */
+  --_secondary-border-width: var(--rh-border-width-sm, 1px);
+  --_secondary-active-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-background-color: transparent;
+  --_secondary-active-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-focus-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-background-color: transparent;
+  --_secondary-focus-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-hover-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-background-color: transparent;
+  --_secondary-hover-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* TERTIARY */
+  --_tertiary-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-background-color: transparent;
+  --_tertiary-border-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_tertiary-border-width: var(--rh-border-width-sm, 1px);
+  --_tertiary-active-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-active-background-color: transparent;
+  --_tertiary-active-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-active-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-focus-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-focus-background-color: transparent;
+  --_tertiary-focus-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-hover-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-hover-background-color: transparent;
+  --_tertiary-hover-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* LINK */
+  --_link-color: var(--rh-color-interactive-blue-lighter, #73bcf7);
+  --_link-background-color: transparent;
+  --_link-active-color: var(--rh-color-interactive-blue-lightest, #bee1f4);
+  --_link-active-background-color: transparent;
+  --_link-focus-color: var(--rh-color-interactive-blue-lightest, #bee1f4);
+  --_link-focus-background-color: transparent;
+  --_link-hover-color: var(--rh-color-interactive-blue-lightest, #bee1f4);
+  --_link-hover-background-color: transparent;
+
+  /* CLOSE */
+  --_close-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_close-background-color: transparent;
+
+  /* PLAY */
+  --_play-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_play-background-color: #505050; /* WARNING: not a token value */
```

Comment:
`--_play-background-color: var(--rh-color-white, #ffffff);`
`--_play-background-opacity:  var(--rh-opacity-30, 30%);`
`--_play-active-background-color: var(--rh-color-white, #ffffff);`
`--_play-active-background-opacity: var(--rh-opacity-50, 50%);`
`--_play-focus-background-color: var(--rh-color-white, #ffffff);`
`--_play-focus-background-opacity: var(--rh-opacity-50, 50%);`
`--_play-hover-background-color: var(--rh-color-white, #ffffff);`
`--_play-hover-background-opacity: var(--rh-opacity-50, 50%);`

Reply:
the opacity values will be difficult to implement with hex-based tokens

Reply:
I think I would rather see us use HSL than go down this path but: https://caniuse.com/?search=hex%20alpha I was surprised there was this much browser support for alpha on hex format.  And I do realize that the way our tokens currently work even combing the alpha % won't work with the hex without string manipulation.  Is this an opportunity to revisit HSL in tokens?

Reply:
ideally, style-dictionary would let us automatically transform the inputs to rgb / hsl values

IN:
```yaml
color:
  blue:
    500:
      $value: '#0066cc'
```
OUT:
```css
:root {
  --rh-color-blue-500: #0066cc;
  --rh-color-blue-500-rgb: 0, 102, 204;
  --rh-color-blue-500-hsl: 210 100% 40%;
}
```
USAGE:
```css
#play-button {
  background-color: hsl(var(--rh-color-blue-500-hsl) / var(--rh-opacity-50));
}
```

[codepen](https://codepen.io/bennyp/pen/vYamYGw?editors=1100)

*BUT* this is non-trivial, since style-dictionary 3 doesn't support splitting inputs into multiple outputs.

Reply:
well, i whipped something up in the tokens repo that should help with this.

Reply:
@marionnegp PTAL these should be ready to go